Userland/Libraries/LibJS/Runtime/ArrayBufferPrototype.cpp

```diff
@@ -37,51 +38,96 @@ ArrayBufferPrototype::~ArrayBufferPrototype()
 // 25.1.5.3 ArrayBuffer.prototype.slice ( start, end ), https://tc39.es/ecma262/#sec-arraybuffer.prototype.slice
 JS_DEFINE_NATIVE_FUNCTION(ArrayBufferPrototype::slice)
 {
+    // 1. Let O be the this value.
+    // 2. Perform ? RequireInternalSlot(O, [[ArrayBufferData]]).
     auto* array_buffer_object = TRY(typed_this_value(global_object));
-    // FIXME: Check for shared buffer
+
+    // 3. If IsSharedArrayBuffer(O) is true, throw a TypeError exception.
+    // fixme: shared buffer
```

Comment:
Please capitalise FIXME, and be a bit more verbose - comments should be sentences.

Either:

```
// 3. If IsSharedArrayBuffer(O) is true, throw a TypeError
// FIXME: Handle SharedArrayBuffer.
```

or

```
// FIXME:
// 3. If IsSharedArrayBuffer(O) is true, throw a TypeError
```

Reply:
I had meant to replace this for `// fixme: check for shared buffer` as elsewhere, I'll replace with the appropriate capitalisation also :)